### PR TITLE
AlertingNG: use json rather than base64 encoded data frames

### PR DESCRIPTION
**Pull request by ryantxu**
This replaces the base64 encoded response with JSON

:warning: totally untested!

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
LGTM :man_shrugging:

File: pkg/services/ngalert/api/api.go [hunk 1]
```diff
@@ -130,14 +129,9 @@ func (api *API) conditionEvalEndpoint(c *models.ReqContext, cmd ngmodels.EvalAle
...
-	df := plugins.NewDecodedDataFrames([]*data.Frame{&frame})
-	instances, err := df.Encoded()
-	if err != nil {
-		return response.Error(400, "Failed to encode result dataframes", err)
-	}
 
-	return response.JSON(200, util.DynMap{
-		"instances": instances,
+	return response.JSONStreaming(200, util.DynMap{
+		"instances": []*data.Frame{&frame},
```
Comment (ryantxu) [inline pkg/services/ngalert/api/api.go:134]: does this need/want to be an array?  in ever case it looks like it is just a single instance

**Review by torkelo — APPROVED** [review]
I took this for a spin but these parts are not working in master either so not sure where things stand with the alerting NG UI

**Comment by torkelo** [comment]
IE I cannot eval & see any results / table in the alert ng view in master (nor in this branch)

**Comment by ryantxu** [comment]
I'll wrap this into the arrow PR directly, and when the alertingNG UI gets some attention, we can fix any issues this may have caused

**Pull request closed without merging**